zcash_client_backend/src/data_api/error.rs

```diff
@@ -34,6 +35,12 @@ pub enum Error<NoteId> {
     /// Chain validation detected an error in the block at the specified block height.
     InvalidChain(BlockHeight, ChainInvalid),
 
+    /// A provided extsk is not associated with the specified account.
+    AccountNotFound(AccountId),
+
+    /// No key of the given type was associated with the specified account.
+    KeyNotFound(AccountId, Typecode),
+
```

Comment:
These are maximally non-sorted 😝 I'd prefer they be sorted into the (currently mostly sorted) list of enum cases.